raiden/network/matrixtransport.py

```diff
@@ -204,6 +205,8 @@ def start_health_check(self, node_address):
             # Ensure network state is updated in case we already know about the user presences
             # representing the target node
             self._update_address_presence(node_address)
+            assert self._get_room_for_address(node_address, allow_missing_peers=True),\
```

Comment:
the slash at the end of the line in unnecessary, additionally `assert`s are IMO more readable if they use only one line

Reply:
Without the slash, `assert <expression>,` fails with `SyntaxError`. backslash is to break this big line in comprehensible parts